archive/migrator/src/migrator.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+Migrator lambda
+
+Receives notifications from the bagger service and triggers
+archive ingest operations.
+"""
+
+import json
+import os
+
+from botocore.vendored import requests
+
+
+def archive_bag_api_messages(bags, bucket):
+    for bag in bags:
+        yield {
+            "type": "Ingest",
+            "ingestType": {"id": "create", "type": "IngestType"},
+            "space": {"id": "wellcome-test", "type": "Space"},
+            "uploadUrl": f"s3://{bucket}/{bag}",
+        }
+
+
+def print_result(description, result):
+    print(description)
+    dumped_json = json.dumps(result.json(), indent=2)
+    print(dumped_json)
+
+
+def call_ingest_api(bucket_name, bags, api):
+    print(api)
+    session = requests.Session()
+
+    for message in archive_bag_api_messages(bags, bucket_name):
+        response = session.post(api, json=message)
+        status_code = response.status_code
+
+        if status_code != 201:
+            print_result(f"ERROR calling {api}", response)
+        else:
+            print(f"{message} -> {api} [{status_code}]")
+            location = response.headers.get("Location")
```

Comment:
Don't we always expect the Location header to be present? Should we error here if it's absent?